### PR TITLE
Charge Customer Profile

### PR DESCRIPTION
**Pull request by rhimmelbauer**
Notes:

- Added Unit test for Charge a Customer Profile for both AuthorizeNet and Stripe Processors

### CODE REVIEW

File: src/vendor/processors/stripe_processor.py [hunk 1]
```diff
@@ -1253,3 +1258,35 @@ def subscription_payment(self, subscription):
...
+        
+        stripe_line_items = []
+        for order_item in self.invoice.get_one_time_transaction_order_items():
+            line_item_data = self.build_invoice_line_item(order_item, stripe_invoice.id)
+            stripe_line_item = self.stripe_create_object(self.stripe.InvoiceItem, line_item_data)
+            stripe_line_items.append(stripe_line_item)
+        
+        stripe_invoice.lines = stripe_line_items
+
+        amount = self.convert_decimal_to_integer(self.invoice.get_one_time_transaction_total())
```
Comment (mgcamisa) [inline src/vendor/processors/stripe_processor.py:1282]: just curious why we're converting to integer here- does stripe not deal with cents or something?

Comment (rhimmelbauer) [reply src/vendor/processors/stripe_processor.py:1282]: yes it does but in a weird way

File: src/vendor/processors/authorizenet.py [hunk 2]
```diff
@@ -849,6 +883,60 @@ def get_customer_id_for_expiring_cards(self, month):
 
         return customer_profile_ids
     
+    def get_customer_and_payemnt_id_for_expiring_cards(self, month):
```
Comment (mgcamisa) [inline src/vendor/processors/authorizenet.py:886]: typo: *get_customer_and_payment_id_for_expiring_cards

here and in test_authorizenet

**Review by mgcamisa — APPROVED** [review]

File: src/vendor/processors/authorizenet.py (hunk 2 above)
Comment (rhimmelbauer) [reply src/vendor/processors/authorizenet.py:886]: thank you !!!!

**Pull request merged**